code/modules/reagents/chemistry/chem_holder.dm

```diff
@@ -0,0 +1,524 @@
+//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
+
+var/const/TOUCH = 1 //splashing
+var/const/INGEST = 2 //ingestion
+var/const/VAPOR = 3 //foam, spray, blob attack
+var/const/PATCH = 4 //patches
+var/const/INJECT = 5 //injection
+
+///////////////////////////////////////////////////////////////////////////////////
+
+/datum/reagents
+	var/list/reagents = new/list()
```

Comment:
datum/reagents should really be datum/reagent_holder or something, but it'd be hell to change.